src/common/components/AggregationBox.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Col, Row } from 'antd';
+
+class AggregationBox extends Component {
+  render() {
+    const { name, action } = this.props;
+
+    return (
+      <div className="w-100 mb3 br1">
+        <Row type="flex" justify="space-between">
+          <Col>
+            <h3>{name}</h3>
+          </Col>
+          <Col>{action}</Col>
+        </Row>
+        <Row className="w-100">{this.props.children}</Row>
+      </div>
+    );
+  }
+}
+
+AggregationBox.propTypes = {
+  action: PropTypes.node,
```

Comment:
maybe calling it `headerAction` would be more obvious.

Reply:
each component has each one scope and layout, IMO is to specific